vilebot/src/main/java/com/oldterns/vilebot/handlers/user/QuotesAndFacts.java

```diff
@@ -428,4 +447,97 @@ private static String trimChars( String input, String charsToRemove )
 
         return new String( input.substring( st, len ) );
     }
+
+    private static void saveToFile( Set<String> data, String queried, String sender, String mode, ReceivePrivmsg event,
+                                    String overWrite )
+    {
+        if ( !DUMP_FOLDER_FILE.exists() )
+        {
+            DUMP_FOLDER_FILE.mkdirs();
+        }
+
+        File dumpFile = new File( DUMP_FOLDER_FILE, mode + "-of-" + queried + "-from-" + sender );
+
+        if ( dumpFile.exists() && !overWrite.equals( "ow" ) )
```

Comment:
I'm not sure I understand the utility of overwrite. If file exists, can't you just use it? Or is this meant to capture the instance where quotes/facts are added before the garbage collector (or cron job as the case may be) comes around?

Reply:
Yep the latter is basically the reason for it. The options were have the file always overwrite for no risk of missing info, have an overwrite option for it only writes to file if told to/if the file doesn't exist, or compare the file with what was returned to see if it matches. I figured the overwrite would be the best option of the three